<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationPage</class>
 <widget class="QWidget" name="ApplicationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure the application&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="singleApplicationCheckBox">
     <property name="toolTip">
      <string>Select, if only one instance of the application should be running</string>
     </property>
     <property name="text">
      <string>Single Application Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="splashScreenCheckBox">
     <property name="toolTip">
      <string>Select to show the startup splash screen</string>
     </property>
     <property name="text">
      <string>Show Splash Screen at startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Open at startup</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="noOpenRadioButton">
        <property name="toolTip">
         <string>Select to not open anything</string>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="lastFileRadioButton">
        <property name="toolTip">
         <string>Select to open the most recently opened file</string>
        </property>
        <property name="text">
         <string>Last File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="lastProjectRadioButton">
        <property name="toolTip">
         <string>Select to open the most recently opened project</string>
        </property>
        <property name="text">
         <string>Last Project</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="lastMultiprojectRadioButton">
        <property name="toolTip">
         <string>Select to open the most recently opened multiproject</string>
        </property>
        <property name="text">
         <string>Last Multiproject</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="globalSessionRadioButton">
        <property name="toolTip">
         <string>Select to restore the global session</string>
        </property>
        <property name="text">
         <string>Global Session</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Check for updates</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QRadioButton" name="noCheckRadioButton">
        <property name="toolTip">
         <string>Select to disable update checking</string>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="alwaysCheckRadioButton">
        <property name="toolTip">
         <string>Select to check for updates at every startup</string>
        </property>
        <property name="text">
         <string>Always</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dailyCheckRadioButton">
        <property name="toolTip">
         <string>Select to check for updates once a day</string>
        </property>
        <property name="text">
         <string>Daily</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="weeklyCheckRadioButton">
        <property name="toolTip">
         <string>Select to check for updates once a week</string>
        </property>
        <property name="text">
         <string>Weekly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="monthlyCheckRadioButton">
        <property name="toolTip">
         <string>Select to check for updates once a month</string>
        </property>
        <property name="text">
         <string>Monthly</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Reporting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="systemEmailClientCheckBox">
        <property name="toolTip">
         <string>Select to use the system email client to send reports</string>
        </property>
        <property name="text">
         <string>Use System Email Client</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Error Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="errorlogCheckBox">
        <property name="toolTip">
         <string>Select to check the existence of an error log upon startup</string>
        </property>
        <property name="text">
         <string>Check for Error Log at Startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>571</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>singleApplicationCheckBox</tabstop>
  <tabstop>splashScreenCheckBox</tabstop>
  <tabstop>noOpenRadioButton</tabstop>
  <tabstop>lastFileRadioButton</tabstop>
  <tabstop>lastProjectRadioButton</tabstop>
  <tabstop>lastMultiprojectRadioButton</tabstop>
  <tabstop>globalSessionRadioButton</tabstop>
  <tabstop>noCheckRadioButton</tabstop>
  <tabstop>alwaysCheckRadioButton</tabstop>
  <tabstop>dailyCheckRadioButton</tabstop>
  <tabstop>weeklyCheckRadioButton</tabstop>
  <tabstop>monthlyCheckRadioButton</tabstop>
  <tabstop>systemEmailClientCheckBox</tabstop>
  <tabstop>errorlogCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
